.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
```

Comment:
I suggest supporting teams with Node.js version>14 inclusive. Why? Otherwise we 'lose customers'. Node.js 14 does not [support many 2021 features](https://node.green/#ES2020). Consequently, I would assign es2020.

Yes, we can support different eslint configurations in root vs generated-code, but maybe for the sake of simplicity we should stick to one for all?